library_generation/generate_composed_library.py

```diff
@@ -54,8 +54,8 @@ def generate_composed_library(
     :param library_path: the path to which the generated file goes
     :param library: a LibraryConfig object contained inside config, passed here
     for convenience and to prevent all libraries to be processed
-    :param output_folder:
-    :param versions_file:
+    :param output_folder: the folder to where tools go
```

Comment:
Later, we should consider renaming this folder because it is not where we create the resulting library/repo (which is the actual output of the script (?))

Reply:
I agree, the name is a bit of confusing.

Do you have a better name in mind?

Reply:
Although anything created by the script is output, in a functional perspective the output we care about is the one we have a final use for. Just clarifying this rationale because "anything is output" can be valid too (and this is more of a suggestion in the end).

Reply:
Thanks. I'll keep it as-is for now (I need sometime to think of a better name). 

We can come up with a new name later.

Reply:
> I agree, the name is a bit of confusing.
> 
> Do you have a better name in mind?

I find it slightly tough to answer. What about `workspace`?

Reply:
Yeah, this sounds good.